python/lsst/sims/utils/samplingFunctions.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from lsst.sims.utils import ObservationMetaData
+
+__all__ = ['sample_obsmetadata', 'samplePatchOnSphere']
+
+def sample_obsmetadata(obsmetadata, size=1, seed=1):
+    """
+    Sample a square patch on the sphere overlapping obsmetadata
+    field of view by picking the area enclosed in
+    obsmetadata.pointingRA \pm obsmetadata.boundLength
+    obsmetadata.pointingDec \pm obsmetadata.boundLength
+
+    Parameters
+    ----------
+    obsmetadata: instance of
+        `sims.catalogs.generation.db.ObservationMetaData`
+    size: integer, optional, defaults to 1
+        number of samples
+
+    seed: integer, optional, defaults to 1
+        Random Seed used in generating random values 
+    Returns
+    -------
+    tuple of ravals, decvalues
+    """
+
+    mydict = obsmetadata.summary
+    phi = np.radians(mydict['pointingRA'])
+    theta = np.radians(mydict['pointingDec'])
+    equalrange = np.radians(mydict['boundLength'])
+    ravals, thetavals = samplePatchOnSphere(phi=phi,
+					    theta=theta,
+					    delta=equalrange,
+					    size=size,
+                                            seed=seed)
+    return ravals, thetavals
+
+
+def samplePatchOnSphere(phi, theta, delta, size, seed=1):
+    """
+    Uniformly distributes samples on a patch on a sphere between phi \pm delta,
+    and theta \pm delta on a sphere. Here, the coordinate system is the usual
+    spherical coordinate system but with the azimuthal angle theta going from
+    pi/2.0 at the North Pole, to - pi/2.0 at the South Pole, through 0. at the
+    equator. This function does not work at the poles. The region must not go
```

Comment:
If the function does not work near the poles, add a test at the beginning that will raise an exception if the user specifies invalid input.